### ISSUE TITLE
Error using python 3.9

### ISSUE THREAD

**Issue by randyarrasmith** [dieselrabbit/screenlogicpy#15]
screenlogicpy works great in python 3.7 (on my desktop)

On my laptop which has python 3.9, I am getting this error:
Traceback (most recent call last):
  File "/Users/randy/mypool/bin/screenlogicpy", line 8, in <module>
    sys.exit(cli())
TypeError: cli() missing 1 required positional argument: 'cli_args'

Steps to reproduce (assuming you have 3.9 on system):
1) python3 -m venv mypool
2) . mypool/bin/activate
3) pip install screenlogicpy
4) screenlogicpy

**Comment by nash-an** [comment]
Getting the same issue on Python 3.9

```cmd
C:\Users\Nash\AppData\Local\Programs\Python\Python39\Lib\site-packages\screenlogicpy>screenlogicpy
Traceback (most recent call last):
  File "c:\users\nash\appdata\local\programs\python\python39\lib\runpy.py", line 197, in _run_module_as_main
    return _run_code(code, main_globals, None,
  File "c:\users\nash\appdata\local\programs\python\python39\lib\runpy.py", line 87, in _run_code
    exec(code, run_globals)
  File "C:\Users\Nash\AppData\Local\Programs\Python\Python39\Scripts\screenlogicpy.exe\__main__.py", line 7, in <module>
TypeError: cli() missing 1 required positional argument: 'cli_args'

C:\Users\Nash\AppData\Local\Programs\Python\Python39\Lib\site-packages\screenlogicpy>
```

This also happens with arguments such as -h, -v, and discover.

Steps to reproduce (Py3.9):

1. `pip install screenlogicpy`
2. `cd C:\Users\Nash\AppData\Local\Programs\Python\Python39\Lib\site-packages\screenlogicpy`
3. `screenlogicpy`

**Comment by dieselrabbit** [comment]
Unfortunately, I haven't been able to reproduce this with Python 3.9.7

Not sure where to go from here.

**Comment by nash-an** [comment]
> Unfortunately, I haven't been able to reproduce this with Python 3.9.7
> 
> Not sure where to go from here.

Just did a clean installation of 3.9.7 (windows 10), then installed screenlogic py (python -m pip install screenlogicpy)
Same issue before

I'll try to see if I can find the issue on my own machine, but I doubt it

**Comment by dieselrabbit** [comment]
I think I know what the problem is.

To confirm, can you try running as a module? (python -m screenlogicpy.cli)

**Comment by nash-an** [comment]
Running as a module does seem to work.
```
Discovered 'Pentair: ##-##-##' at 192.168.0.100:80
EasyTouch2 8
**************************
Pool temperature is last 78°F
Pool Heat Set Point: 85°F
Pool Heat: Off
Pool Heat Mode: Off
--------------------------
Spa temperature is last 100°F
Spa Heat Set Point: 100°F
Spa Heat: Off
Spa Heat Mode: Off
--------------------------
**************************
 ID  STATE  NAME
--------------------------
500    Off  Spa
501     On  Water Light
502    Off  Air Blower
503    Off  Water Feature
504    Off  Aux 4
505    Off  Pool
506    Off  Aux 5
507    Off  Aux 6
508    Off  Aux 7
510    Off  Spillway
511    Off  Cleaner
512    Off  Feature 3
513    Off  Feature 4
514    Off  Feature 5
515    Off  Feature 6
516    Off  Feature 7
517    Off  Feature 8
519    Off  AuxEx
**************************
```

**Comment by dieselrabbit** [comment]
Thanks. I think that confirms I screwed up the console script entry point. I should have a fix in the next release.

**Comment by dieselrabbit** [comment]
Fixed with d13e93b8cd52fb238c0f24c74402b1120e094e13
Thank you for your patience.

**Comment by randyarrasmith** [comment]
no, no... thank YOU.   I appreciate you making and maintaining this package and love that it is integrated in Home Assistant.
